grommet/grommet
# Add hover indicator to control button when menu opens

#### What does this PR do?
This PR adds hover styling to the menu control button when it is included in the menu drop. Initially, this change was going to be included in a different Menu PR, but for clarity, I separated it into its own PR.

#### Where should the reviewer start?
src/js/components/Menu/Menu.js

#### What testing has been done on this PR?
This has been testing in local storybook examples by opening the menu then hovering over the control button with the mouse to check that it receives the same hover indicator as the menu items.

#### How should this be manually tested?
By using the storybook examples where the control button is included in the drop (such as "Custom"), open the menu and hover over the control button. It should receive the same hover indicator that the menu items receive.

#### Any background context you want to provide?
The control button was not receiving any styling when it was being hovered over even though it does receive styling when given keyboard focus. For clarity and consistency of styling when the control button is included within the menu drop, it should receive a hover indicator when the mouse hovers over.

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
This introduces new styling on the control button when hovered, but it does not affect any of the functionality of the button.
